storage/tools/dump_tree/nochange_test.go

```diff
@@ -74,23 +75,22 @@ func TestDBFormatNoChange(t *testing.T) {
 		if err != nil {
 			t.Fatalf("ReadFile(%v): %v", tc.file, err)
 		}
-
-		savedS := strings.Split(string(saved), "\n\n")
-		outS := strings.Split(out, "\n\n")
-		if got, want := len(outS), len(savedS); got != want {
-			t.Fatalf("%v dump_tree: got %v lines, want %v", tc.desc, got, want)
+		outT := parseTiles(t, out)
```

Comment:
how about calling this "got" and savedT "want"?

Reply:
Done.